content/en/serverless/agent_configuration/_index.md

```diff
@@ -0,0 +1,138 @@
+| `DD_APM_REPLACE_TAGS`     | Defines a set of rules to replace or remove certain resources, tags containing potentially sensitive [information][3].                                                                                                                |
+| `DD_APM_IGNORE_RESOURCES` | An exclusion list of regular expressions can be provided to disable certain traces based on their resource name all entries must be surrounded by double quotes and separated by commas.                                              |
+| `DD_APM_LOG_FILE`         | The full path to the file where APM-agent logs are written.                                                                                                                                                                           |
+| `DD_APM_LOG_THROTTLING`   | Limits the total number of warnings and errors to 10 for every 10 second interval.                                                                                                                                                    |
+| `DD_APM_CONNECTION_LIMIT` | The APM [connection limit][4] for the Agent.                                                                                                                                                                                          |
+
+### Advanced networking configuration
+
+| Env Variable             | Description                                                                                                                                                                                                                                                                                                                        |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `DD_SKIP_SSL_VALIDATION` | Setting this option to "true" tells the Agent to skip validation of SSL/TLS certificates. Defaults: false                                                                                                                                                                                                                          |
+| `DD_SSLKEYLOGFILE`       | Specifies a destination for TLS master secrets in NSS key log format to allow external programs such as Wireshark to decrypt TLS connections. For more details, see https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/Key_Log_Format. Use of sslkeylogfile compromises security and should only be used for debugging. |
```

Comment:
Not sure if there is an easy way for customers to use this one inside lambda either